src/main/java/com/solvd/webtest/pages/ios/CategoriesPageIos.java

```diff
@@ -0,0 +1,24 @@
+package com.solvd.webtest.pages.ios;
+
+import com.solvd.webtest.pages.base.CategoriesPageBase;
+import com.zebrunner.carina.utils.factory.DeviceType;
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+@DeviceType(pageType = DeviceType.Type.IOS_PHONE, parentClass = CategoriesPageBase.class)
+public class CategoriesPageIos extends CategoriesPageBase {
+    @FindBy(css = "span.cat-title , .l1-wrapper >.cat-container > li > .cat-title")
```

Comment:
It's not a stable selector. What if an additional div will be added inside .l1-wrapper?

Reply:
I don`t think there is better locator

Reply:
It finds names of categories